<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(0, 85, 127)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>28</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 85, 127)
font color white;</string>
    </property>
    <property name="text">
     <string>   ATM</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>481</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(245, 245, 245)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>ATM</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonLogin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>Kirjaudu sisään</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSulje">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>Sulje</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>311</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TERVETULOA! &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;KIRJAUDU SISÄÄN ALOITTAAKSESI.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLogout">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>Kirjaudu ulos</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tili</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonTili">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>Näytä saldo</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSaldo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>361</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(245, 245, 245)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTapah">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>Tapahtumat</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTapah">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>361</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(245, 245, 245)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Nosto</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonNosto20">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>20€</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNosto">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNosto40">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>40€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNosto60">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>60€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNosto500">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>500€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNosto200">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>200€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNosto100">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>100€</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Talletus</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonTalletus20">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>20€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTalletus40">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>40€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTalletus60">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>60€</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTalletus">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTalletus500">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>500€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTalletus200">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>200€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTalletus100">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 170, 0)</string>
      </property>
      <property name="text">
       <string>100€</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
